packages/react-saasify/src/components/NavHeader/styles.module.css

```diff
@@ -88,31 +97,92 @@
 
 .actions {
   display: flex;
+  margin-left: auto;
 }
 
-@media only screen and (max-width: 767px) {
-  .logo {
-    margin-right: 0;
-  }
+.actions .link {
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+.actions .link:first-of-type {
+  margin-left: 0;
+}
+
+.actions .link:last-of-type {
+  margin-right: 0;
+}
 
+.burger {
+  display: none;
+}
+
+@media only screen and (max-width: 767px) {
   .links {
+    display: block;
+    margin-top: 24px;
+    margin-bottom: 8px;
     width: 100%;
-    justify-content: space-around;
   }
 
   .link {
     padding: 0;
     margin-left: 0.2em;
-    font-size: 1em;
+    margin-bottom: 8px;
+    font-size: 16px;
   }
 
   .actions {
     width: 100%;
-    justify-content: space-around;
     margin-top: 0.5em;
   }
 
+  .actions .link {
+    width: 100%;
+  }
+
+  .actions .link:first-of-type {
+    margin-left: 0px;
```

Comment:
(nit) i try and use `0` instead of adding a unit to it because it saves some bytes and does the same thing.